keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt

```diff
@@ -153,33 +153,14 @@ class ImageService(
     }
   }
 
-  suspend fun findBaseAmiName(baseImageName: String): String {
-    return cloudDriverService.namedImages(DEFAULT_SERVICE_ACCOUNT, baseImageName, "test")
-      .lastOrNull()
-      ?.let { namedImage ->
-        findBaseAmiName(namedImage)
-      } ?: throw BaseAmiNotFound(baseImageName)
-  }
-
-  suspend fun findBaseAmiName(baseImageId: String, region: String): String {
-    return cloudDriverService.getImage("test", region, baseImageId)
-      .lastOrNull()
-      ?.let { namedImage ->
-        findBaseAmiName(namedImage)
-      } ?: throw BaseAmiNotFound(baseImageId)
-  }
-
   /**
-   * Extracts the base ami version from the tags of a named image
+   * Given a [baseImageId] this function returns its name.
    */
-  private fun findBaseAmiName(image: NamedImage): String? =
-    image
-      .tagsByImageId
-      .values
-      .filterNotNull()
-      .find { it.containsKey("base_ami_name") }
-      ?.getValue("base_ami_name")
-
+  suspend fun findBaseImageName(baseImageId: String, region: String): String =
+    cloudDriverService.getImage("test", region, baseImageId)
+      .lastOrNull()
+      ?.imageName
+      ?: throw BaseAmiNotFound(baseImageId)
```

Comment:
This is the base image we're looking up, so we just want its name, not its own base image name

Reply:
I know it's not directly related to this PR, but I figure I'll ask since I'm looking: do you know why it is that we hard-code `"test"` as the account here?

Reply:
Because images are always baked it test